tutorials/fundamentals/regex-no-start.md

```diff
@@ -0,0 +1,7 @@
+To check if a string does not start with a regular expression, negate `startsWith()` while checking for a regular expression.
```

Comment:
Same with other tutorial. `startsWith()` doesn't work. You want to use either:

```javascript
function doesNotStartWithA(str) {
  return !/^a/.test(str);
}

// or:
function doesNotStartWithA(str) {
  return /^[^a]/.test(str);
}
```

Where `[^a]` matches anything that is not the character 'a'